src/scene-entry-manager.js

```diff
@@ -458,10 +458,11 @@ export default class SceneEntryManager {
       if (entry.type === "scene_listing" && this.hubChannel.permissions.update_hub) return;
 
       // If user has HMD lifted up, delay spawning for now. eventually show a modal
-      const delaySpawn = isIn2DInterstitial() && !isMobileVR;
+      const spawnDelay = isIn2DInterstitial() ? (isMobileVR ? 1000 : 3000) : 0;
```

Comment:
If I recall correctly the problem was that things would spawn up in the air above the user if their headset was on their forehead. If that's the case, fixing the spawn location seems like a better solution than adding a delay